br/pkg/task/backup.go

```diff
@@ -324,6 +360,10 @@ func RunBackup(c context.Context, g glue.Glue, cmdName string, cfg *BackupConfig
 	if err != nil {
 		return errors.Trace(err)
 	}
+
+	// set cipher only for checkpoint
+	client.SetCipher(&cfg.CipherInfo)
```

Comment:
why need this?

Reply:
encrypt the key info in the checkpoint files if in need